kiva/agg/tests/affine_matrix_test_case.py

```diff
@@ -86,7 +86,7 @@ def test_multiply(self):
 
     def test_determinant(self):
         orig = array((1.0,2.0,3.0,1.0,4.0,5.0))
-        desired = -0.2
+        desired = -5.0
```

Comment:
Crazy. Looks like the matrix inversion code also changed, which explains why none of the other unit tests broke...